### PR TITLE
Added kick reason and JSDoc to Player entity kick method type

### PR DESCRIPTION
**Pull request by BKarner**

### CODE REVIEW

File: server/index.d.ts [hunk 1]
```diff
@@ -232,7 +232,12 @@ declare module "alt-server" {
 
     public giveWeapon(weaponHash: number, ammo: number, equipNow: boolean): void;
 
-    public kick(): void;
+    /**
+     * Forcefully disconnects the player from the server with an error message.
+     *
+     * @param reason The reason to display to the player on the disconnect screen.
```
Comment (Lhoerion) [inline server/index.d.ts:238]: I think it should be more like
`The reason that will be displayed on the disconnect screen.`
or
`The reason that will display to the player on the disconnect screen.`

File: server/index.d.ts [hunk 2]
```diff
@@ -232,7 +232,12 @@ declare module "alt-server" {
 
     public giveWeapon(weaponHash: number, ammo: number, equipNow: boolean): void;
 
-    public kick(): void;
+    /**
+     * Forcefully disconnects the player from the server with an error message.
```
Comment (Lhoerion) [inline server/index.d.ts:236]: It ain't an error message, so transform the sentence or remove this part.

File: server/index.d.ts (hunk 1 above)
Comment (BKarner) [reply server/index.d.ts:238]: Changed it to the latter

File: server/index.d.ts (hunk 2 above)
Comment (BKarner) [reply server/index.d.ts]: Changed it to "reason message" to match the parameter

File: server/index.d.ts [hunk 3]
```diff
@@ -232,7 +232,12 @@ declare module "alt-server" {
 
     public giveWeapon(weaponHash: number, ammo: number, equipNow: boolean): void;
 
-    public kick(): void;
+    /**
+     * Forcefully disconnects the player from with a reason message.
```
Comment (Lhoerion) [inline server/index.d.ts:236]: There's a mistake here, lack of `the server` words between `from with`
edit: Or actually `from` word is obsolete there

**Review by Lhoerion — CHANGES_REQUESTED** [review]

File: server/index.d.ts (hunk 3 above)
Comment (BKarner) [reply server/index.d.ts:236]: Oh god yeah. Whoops

**Review by Lhoerion — APPROVED** [review]
Thanks for contribution, indeed I missed at some point that `kick` method lacks an parameter

**Pull request merged**